src/components/SocialButtons/SocialButtons.tsx

```diff
@@ -0,0 +1,88 @@
+import { FC, useState } from "react";
+
+import { icons } from "../Icon/Icon";
+import {
+  Container,
+  SocialButton,
+  StyledIcon,
+  PlainLink,
+  Divider,
+} from "./SocialButtons.styles";
+
+const destinations: {
+  icon: keyof typeof icons;
+  primary: string;
+  locales?: Record<string, string>;
+}[] = [
+  {
+    icon: "twitter",
+    primary: "https://twitter.com/airswap",
+    locales: {
+      en: "https://twitter.com/airswap",
+      fr: "https://twitter.com/airswap_fr",
+    },
+  },
+  {
+    icon: "discord",
+    primary: "https://chat.airswap.io/",
+  },
+  {
+    icon: "medium",
+    primary: "https://blog.airswap.io/",
+  },
+  {
+    icon: "telegram",
+    primary: "https://t.me/airswapofficial",
+  },
+];
+
+const SocialButtons: FC<{}> = () => {
```

Comment:
you don't really need a generic argument, just `FC` will do. It doesn't really matter, it just keeps my IDE happy ;)